File: C++/FizzBuzz.cpp
#include <iostream>

int main() {
  int a, b, c;
  std::cin >> a >> b >>c;
  for (int i = 1; i <= c; i++ ) {
    if (i%a == 0 && i%b == 0) {
      std::cout << "FizzBuzz\n";
    } else if (i%a == 0) {
      std::cout << "Fizz\n";
    } else if (i%b == 0) {
      std::cout << "Buzz\n";
    } else {
      std::cout << i << "\n";
    }
  }
}

File: Python/FizzBuzz.py
def main():
    e = input().strip().split()
    x,y,z = int(e[0]), int(e[1]), int(e[2])
    for i in range(1, z+1):
        if i%x == 0 and i%y == 0:
            print("FizzBuzz")
        elif i%x == 0:
            print("Fizz")
        elif i%y == 0:
            print("Buzz")
        else:
            print(str(i))
main()
